docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -686,6 +672,32 @@ p2p-static-peers: ["/ip4/151.150.191.80/tcp/9000/p2p/16Ui...aXRz",
 List of comma-separated [multiaddresses](https://docs.libp2p.io/reference/glossary/#multiaddr)
 of static peers.
 
+### rest-api-docs-enabled
```

Comment:
It looks like rest-api-docs-enabled got added instead of rest-api-host-whitelist (but when I look at the file both are there)

Reply:
I had to reorganise content alphabetically, that's probably the cause.